#inputs
bool setSuctionState         # suction ON/OFF
std_msgs/Int16 suction_force # suction force values from 0 (super weak) to 40 (full power)
---  
#outputs
bool succeeded
